src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -2218,6 +2237,7 @@ because xbuild doesn't support framework reference assemblies.
     EnablePreloadAssembliesDefault="$(_AndroidEnablePreloadAssembliesDefault)"
     PackageNamingPolicy="$(AndroidPackageNamingPolicy)"
     BoundExceptionType="$(AndroidBoundExceptionType)"
+	InstantRunEnabled="$(_InstantRunEnabled)"
```

Comment:
Looks like the formatting is off here.

Reply:
This entire file is formatted in weird ways :P